scripts/nbconverted/predictive_genes.py

```diff
@@ -64,23 +79,23 @@
 X = pd.merge(estimated_latent_df, obsv_df, left_index=True, right_index=True)
 
 # Residuals: observed - estimated
-X['residuals'] = X['observed']-X['estimated']
+X['residuals_latent'] = abs(X['observed']-X['estimated'])
 X.head(5)
 
 # Mean and stdev of residuals
-residual_mean = X['residuals'].values.mean()
-residual_std = X['residuals'].values.std()
+#residual_mean = X['residuals'].values.mean()
```

Comment:
what is going on here? why comments?

Reply:
We are still playing around with the metric that we want to use to determine if a gene is "predictive" (see comment below).  So the comments are for different metrics